ush/python/pygfs/task/analysis.py

```diff
@@ -156,17 +151,22 @@ def get_bias_dict(self) -> Dict[str, Any]:
                 obfile = ob['obs bias']['input file']
                 obdir = os.path.dirname(obfile)
                 basename = os.path.basename(obfile)
-                prefix = '.'.join(basename.split('.')[:-2])
-                for file in ['satbias.nc', 'satbias_cov.nc', 'tlapse.txt']:
-                    bfile = f"{prefix}.{file}"
-                    copylist.append([os.path.join(self.task_config.COM_ATMOS_ANALYSIS_PREV, bfile), os.path.join(obdir, bfile)])
-                    # TODO: Why is this specific to ATMOS?
+                prefix = '.'.join(basename.split('.')[:-3])
+                bfile = f"{prefix}.radbcor"
+                copylist.append([os.path.join(self.task_config.COM_ATMOS_ANALYSIS_PREV, bfile), os.path.join(obdir, bfile)])
+                break
```

Comment:
Thank you for working on this. I was wondering if this would copy/tar bias files of one sensor or all the sensors? 
Also, it looks like this only work for atmos radiance bias files from "COM_ATMOS_ANALYSIS_PREV". I am going to do similar work on aerosol bias correction files, so I was wondering how it handles bias files for other components?

Reply:
Yes, @ypwang19 , these changes were only made with consideration of atmospheric DA.  We use `COM_ATMOS_ANALYSIS_PREV` because the late cycle (e.g., `gdas`) is the one which cycles the radiance bias correction.  `gfs` and `enkf` cycles use the radiance bias correction from the previous cycle `gdas`.

We can generalize for aerosols, but doing so changes the scope of issue #2862 and this PR.  Not a problem.  We can edit #2862 and this PR.  If we want to generalize this, what's the preferred approach?   Should we also consider marine DA in the generalization?  How does this generalization fit into @DavidNew-NOAA 's JEDI class work?  I don't know the answer to these questions.  A team meeting with interested may be needed to chart out the best past forward

Reply:
@ypwang19 and @CoryMartin-NOAA , we should revisit this once @DavidNew-NOAA 's PR [#2805](https://github.com/NOAA-EMC/global-workflow/pull/2805) enters g-w `develop`.  #2805 will necessitate changes to this PR.  This refactor could be an opportunity to explore unification / generalization of g-w processing of JEDI bias correction files.  How does this sound?

Reply:
It sounds good to me. Thanks!